### DISCUSSION TITLE
Flyon UI: Free & Open Source Tailwind CSS Component Library

### DISCUSSION THREAD

**Discussion by abhidave001** [tailwindlabs/tailwindcss#14573] [Show and tell]
Hi All,
I would like to share an open source resource for Tailwind enthusiasts.

Flyon UI is latest free [Tailwind CSS Components Library](https://flyonui.com/). It is designed to combine the best of both worlds: the aesthetic appeal of semantic classes and the powerful functionality of JS plugins.

It has over 78+ components & examples. Additionally, it uses the strength of DaisyUI & Preline. 
Apart from that it is easy to use.

**Features:**

- 800+ Free Components & Examples
- Universal Framework Compatibility
- Unlimited Themes
- Unstyled & Accessible Plugins
- Responsive & RTL support
- Free Forever
- Beautiful and Semantic Styling
- Efficiency and Productivity
- Maintainable and Scalable

Check out the GitHub Repo as well: https://github.com/themeselection/flyonui

**Comment by mary598chess** [comment]
Hello,
Flyon UI sounds like a fantastic resource for [florida blue](https://www-floridablue.com) Tailwind enthusiasts! With over 800 free components and universal framework compatibility, it's definitely worth checking out. The combination of semantic classes and JS plugins, along with unlimited themes and responsive support, makes it a powerful tool for developers. Here’s the GitHub repo link: Flyon UI. Enjoy exploring!

Best Regards,
Mary Chess

**Comment by Starfishht** [comment]
Hi Mary,

Thanks for sharing the info about Flyon UI! It really does sound like a great tool, especially with that huge library of components and the flexibility for different frameworks. I love how it's tailored for Tailwind users. I'll definitely be checking out the GitHub repo. Appreciate the recommendation!
